spec.md

```diff
@@ -692,3 +696,52 @@ Destructuring binding and assignment are not part of ES6, but all binding positi
 ```js
 interface Pattern <: Node { }
 ```
+
+# Concrete syntax
+
+Whitespace, comments, and the exact character sequences comprising Literal nodes are part of the lexical grammar, but not part of the _abstract_ syntax. Their representation is completely optional.
+
+```js
+interface ConcreteNode <: Node {
+    sourceElements: [ ChildReference | Token | Nontoken ];
```

Comment:
No and no, because `ConcreteNode` extends `Node` (which corresponds to a [syntactic production](http://www.ecma-international.org/ecma-262/6.0/#sec-syntactic-grammar)) while `sourceElements` contains lexical [input elements](http://www.ecma-international.org/ecma-262/6.0/#sec-lexical-and-regexp-grammars), which are _not_ nodes.